sky/skylet/log_lib.py

```diff
@@ -210,7 +210,6 @@ def run_with_log(
 
             subprocess.Popen(
                 daemon_cmd,
-                start_new_session=True,
```

Comment:
Let's add a comment here:
```python
# We do not need to set `start_new_session=True` here, as the daemon
# script will detach itself from the parent process with fork to avoid being
# killed by ray job. See the reason we daemonize the process in
# `sky/skylet/subprocess_daemon.py`
```